### PR TITLE
Verifier.verifyInnerStep test

### PR DESCRIPTION
**Pull request by terminaldweller**
i just wanted to show how im testing it so we could be sure im not running it wrong
i changed the Verifier.verifyInnerStep output so it returns number instead of a bool since i wanted to see who's throwing.besides that, didnt touch any other solidity file.
most of the changes are to the js file.
if you think we need the changes i made to the js file, i can write a clean one.
anyways, like i said i just wanted to show you how im running the test.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -25,6 +23,19 @@ function compile() {
...
     console.log('var runnerABI = ' + JSON.stringify(runnerABI) + '')
+
+    const compilerInput2 = {
+        'language': 'Solidity',
+        'sources': {
+        'scryptFramework.sol': {'content': readFile('contracts/scryptFramework.sol')},
+        'scryptVerifier.sol': {'content': readFile('contracts/scryptVerifier.sol')}
+        }
+    }
+    results = JSON.parse(solc.compileStandard(JSON.stringify(compilerInput2)))
```
Comment (chriseth) [inline test/index.js]: why do you compile it twice?

Comment (chriseth) [reply test/index.js]: Ah sorry, did not see that this is about the verifier :-)

File: test/index.js [hunk 2]
```diff
@@ -129,11 +155,21 @@ async function test() {
 }
 
 async function tryStuff() {
-    var runner = await deployIfNeeded()
+    var ret =  await deployIfNeeded()
```
Comment (chriseth) [inline test/index.js]: You can use `var [runner, verifier] = await deployIfNeeded()` here

File: test/index.js [hunk 3]
```diff
@@ -45,9 +56,10 @@ miner.start()
 */
 // Needs to be done every time: personal.unlockAccount(account)
 
-var account = '0xcf2c80cc7ad6dfc8238e9d154b533bd624feb384'
+var account = "0x1c9bd47cb154e2c6c870034037074c4521519486"
 // Remove if not deployed yet
 var contractAddr = 0//'0xfa7bE2e5aEA20CEE4384b109a91f26EdF5338a68'
+var contractAddr2 = 0
```
Comment (chriseth) [inline test/index.js]: runnerAddr / verifierAddr?

Comment (terminaldweller) [reply test/index.js]: yes. 
sorry for the naming .

File: test/index.js (hunk 2 above)
Comment (terminaldweller) [reply test/index.js]: ah ok. now i know . thanks.

**Comment by terminaldweller** [comment]
like is said, if you think its a test that we need on scrypt, i can just cleanup the mess and add another PR.

**Comment by chriseth** [comment]
Yes, please clean up this PR and then we can merge it. You do not need to create a new PR, just pushing to the branch will update the PR. Also you can rebase and squash the commits at some point (and then force-push).

**Comment by terminaldweller** [comment]
cleaned-up, rebased and force-pushed.

**Review by chriseth — APPROVED** [review]

**Pull request merged**